frontend/src/pages/Home.tsx

```diff
@@ -22,11 +22,9 @@ export const Home = () => {
 
   const handleSuggestClick = async () => {
     try {
-      // const response = await axios.get(
-      //   "https://www.themealdb.com/api/json/v1/1/random.php"
-      // );
-
-      const response = await axios.get(`http://localhost:3000/api/suggestion`);
+      const response = await axios.get(
+         "https://www.themealdb.com/api/json/v1/1/random.php"
+       );
 
```

Comment:
You should not modify the `handleSuggestClick` method. It should use `const response = await axios.get(`http://localhost:3000/api/suggestion`);`